README.md

```diff
@@ -166,11 +174,22 @@ It provides various functionalities for managing OceanBase clusters, tenants, ba
 - [x] Backup and Recovery: Periodically backup data to OSS or NFS destinations, restore data from OSS or NFS.
 - [x] Physical Standby: Restore standby tenant from backup, create empty standby tenant, activate standby tenant to primary, primary-standby switchover.
 - [x] Fault Recovery: Single node fault recovery, cluster-wide fault recovery with IP preservation.
+- [x] Dashboard(GUI): A web-based graphical management tool for OceanBase clusters based on ob-operator.
+
+## Storage Compatibility
 
-The upcoming features include:
+We have tested ob-operator with the following storage solutions:
 
-- [ ] Dashboard: A web-based graphical management tool for OceanBase clusters based on ob-operator.
-- [ ] Enhanced operational task resources: This includes lightweight tasks focused on cluster and tenant management, among other features.
+| Storage Solution       | Tested Version | Compatibility | Notes                                        |
+| ---------------------- | -------------- | ------------- | -------------------------------------------- |
+| local-path-provisioner | 0.0.23         | ✅            | Recommended for development and testing      |
+| Rook CephFS            | v1.6.7         | ❌            | CephFS does not support `fallocate` sys call |
+| Rook RBD (Block)       | v1.6.7         | ✅            |                                              |
+| OpenEBS (cStor)        | v3.6.0         | ✅            |                                              |
+| GlusterFS              | v1.2.0         | ❓            | Requires kernel version >= 5.14              |
+| Longhorn               | v1.6.0         | ✅            |                                              |
+| JuiceFS                | v1.1.2         | ✅            |                                              |
+| NFS                    | v5.5.0         | ✅            | Requires NFS version >= 4.3                  |
```

Comment:
we care about the nfs protocol version, it's required >= 4.2, but still have problem with unit deletion